package.json

```diff
@@ -351,7 +351,7 @@
     "ts-jest": "^29.0.5",
     "ts-loader": "^9.4.2",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.5",
+    "typescript": "^5.0.2",
```

Comment:
No breaking changes after this major upgrade?

Reply:
nope! well, except the one thing.